ivdr/release-notes-5.22.md

```diff
@@ -0,0 +1,42 @@
+# Release notes format IVDR
+This format should be used for the release notes for IVDR releases.
+
+# Release Notes OncoAct
+
+## Purpose & Scope
+These release notes documents the release and known issuers of the medical device software OncoAct.
+
+## Definitions
+- Anomaly : Any condition that deviates from the expected based on requirements specifications, design documents, standards, etc. or from someone's perceptions or experiences. Anomalies may be found during, but not limited to, the review, test, analysis, compilation, or use of the medical device software or applicable documentation. 
+
+## Details of released software
+The software has been developed according to a software development SOP (HMF-PRO-265 Software development life cycle).
+
+Name: OncoAct
+Version: 5.22
+Type of release: Minor
```

Comment:
This should be 'Major', because in IVDR context everything up from changing some formatting and text in the report is considered major. Rest seems fine for now!